packages/block-library/src/columns/editor.scss

```diff
@@ -112,26 +112,31 @@
 	}
 }
 
-// In absence of making the individual columns resizable, we prevent them from being clickable.
-// This makes them less fiddly. @todo: This should be revisited as the interface is refined.
-.wp-block-columns [data-type="core/column"] {
-	pointer-events: none;
-
-	// The empty state of a columns block has the default appenders.
-	// Since those appenders are not blocks, the parent, actual block, appears "hovered" when hovering the appenders.
-	// Because the column shouldn't be hovered as part of this temporary passthrough, we unset the hover style.
-	&.is-hovered {
-		> .editor-block-list__block-edit::before {
-			content: none;
-		}
+/**
+ * Vertical Alignment Preview
+ */
+.are-vertically-aligned-top .block-core-columns {
```

Comment:
Minor: Given that the rules are the same, can we just combine the definitions for parent/child selectors?

```
.are-vertically-aligned-top .block-core-columns,
div.block-core-columns.is-vertically-aligned-top {
  justify-content: flex-start;
}
```

Reply:
Yeh that's a good point! Watch out for specificity but I think the `div` over-qualification should protect us there.

Reply:
Fixed